include/eve/detail/function/simd/x86/make.hpp

```diff
@@ -218,9 +217,9 @@ namespace eve::detail
   {
     if constexpr( !ABI::is_wide_logical )
     {
-      using i_t = typename detail::as_mask_t<ABI::bytes/sizeof(T)>::type;
-      detail::as_mask_t<ABI::bytes/sizeof(T)> that{ !!v ? ~i_t(0) : i_t(0) };
-      return that;
+      using s_t = typename logical<wide<T,S,ABI>>::storage_type;
+      constexpr auto m = typename s_t::type{0};
+      return s_t{ !!v ? ~m : m };
```

Comment:
This got to be fixed as we discussed I think. With a test? :)

Reply:
Yes. I will do that rn